src/hdf5/read_dpr_gpm.py

```diff
@@ -138,7 +139,7 @@ def read_dpr_gpm(fname, seqNumber_offset=None):
     atime_obj = datetime.utcfromtimestamp(atime.item())
     # this will use hour and minute to offset files in serial processing
     if not seqNumber_offset:
-        seqNumber_offset = 100000*np.int(atime_obj.strftime('%H%M'))
+        seqNumber_offset = 100000*int(atime_obj.strftime('%H%M'))
```

Comment:
this was added as new ctest failures made this warning:
```
File "/workdir/test_root/build/bin/read_dpr_gpm.py", line 141, in read_dpr_gpm
    seqNumber_offset = 100000*np.int(atime_obj.strftime('%H%M'))
                              ^^^^^^
  File "/opt/views/view/lib/python3.11/site-packages/numpy/__init__.py", line 324, in __getattr__
    raise AttributeError(__former_attrs__[attr])
AttributeError: module 'numpy' has no attribute 'int'.
`np.int` was a deprecated alias for the builtin `int`. To avoid this error in existing code, use `int` by itself. Doing this will not modify any behavior and is safe.
```